portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,8 +67,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // format date
```

Comment:
small note: comments should be formatted like

// Format date.
// Get input data.

Reply:
got it, thanks!! :)